conda_forge_tick/mamba_solver.py

```diff
@@ -228,6 +247,45 @@ def __exit__(self, exc_type, exc_val, exc_tb):
         self.write()
 
 
+def _get_run_exports(link_tuple):
+    c, pkg, jdata = link_tuple
+
+    with tempfile.TemporaryDirectory() as tmpdir:
+        try:
+            # download
+            subprocess.run(
+                f"cd {tmpdir} && curl -s -L {c}/{pkg} --output {pkg}",
+                shell=True,
+            )
+
+            # unpack and read if it exists
+            if os.path.exists(f"{tmpdir}/{pkg}"):
+                conda_package_handling.api.extract(f"{tmpdir}/{pkg}")
+
+            if pkg.endswith(".tar.bz2"):
```

Comment:
Is this an actual file path? If so you might consider using path tools to split the extension off.

Reply:
Yeah. This is annoying to code around

```
In [3]: import os

In [4]: os.path.splitext("blah.tar.gz")
Out[4]: ('blah.tar', '.gz')
```

I'm happy to try but the if statement above seems clearer, rather than splitting and it it is not `.conda`, splitting again.

Reply:
oof, ok